http/swagger.yml

```diff
@@ -2217,6 +2217,12 @@ paths:
               type: string
             required: true
             description: The ID of the dashboard to update.
+          - in: query
+            name: include
+            required: false
+            schema:
+              type: string
```

Comment:
could we make this an enum so that our client generator generates a type the UI can consume?